actix/src/sec-5-arbiter.md

```diff
@@ -15,89 +15,148 @@ for your actors to run inside. When you call `start()` on your actor it is then
 running inside of the System Arbiter's thread. In many cases, this is all you
 will need for a program using Actix.
 
+While it only uses one thread, it uses the very efficient event loop pattern
+which works well for asynchronous events. To handle synchronous tasks that
+require heavy CPU power, it's better to avoid blocking the event loop and
+instead offload the computation to other threads. For this usecase, read the
+next section and consider using [`SyncArbiter`](./sec-6-sync-arbiter.md).
+
+## The event loop
+
+> If you're familiar with event loops in other paradigms, such as libuv (upon
+> which Node runs) or tokio (upon which Actix is implemented), each Arbiter has
+> a similar processing paradigm.
+
+> Specifically, if you're familiar with how JavaScript handles asynchronicity
+> and how the single-threaded model makes it so JS mostly doesn't need
+> synchronization primitives, this should feel very familiar to you.
+
+One `Arbiter` is in control of one thread with one task queue, and when tasks
+get spawned in an Actor's context or using `Arbiter::spawn`, the task gets
```

Comment:
"when tasks get spawned ..., the task gets queued up ..."
This language seems quite informal and a bit confusing to me. You mention plural tasks, then singular. For an english-second language reader, the usage of "gets queued up" may also be unclear.
It's probably better to say "One `Arbiter` is running on one thread with one task queue. When a task is spawned in an `Actor`'s context or using `Arbiter::spawn`, the task is queued for execution in the `Arbiter`'s thread.

Reply:
In general, the language used should be short, simple English, and it should read almost as a recipe, with clear instructions. Active voice should be used over passive voice as well. An example here is passive is "the cake gets put in the oven by William", active is "William put the cake in the oven". I hope this helps,

Reply:
Will address.